cmd/pingdom-exporter/main.go

```diff
@@ -84,6 +86,8 @@ func init() {
 	flag.IntVar(&port, "port", 9158, "port to listen on")
 	flag.IntVar(&outageCheckPeriod, "outage-check-period", 7, "time (in days) in which to retrieve outage data from the Pingdom API")
 	flag.Float64Var(&defaultUptimeSLO, "default-uptime-slo", 99.0, "default uptime SLO to be used when the check doesn't provide a uptime SLO tag (i.e. uptime_slo_999 to 99.9% uptime SLO)")
+	flag.StringVar(&metricsPath, "web.telemetry-path", "/metrics", "Path under which to expose metrics.")
```

Comment:
I'd change the flag format to make it consistent with the other flags.

```suggestion
	flag.StringVar(&metricsPath, "metrics-path", "/metrics", "path under which to expose metrics.")
```